arch/arm/slide_neon.c

```diff
@@ -0,0 +1,48 @@
+/* slide_neon.c -- Optimized hash table shifting for ARM with support for NEON instructions
+ * Copyright (C) 2017 Mika T. Lindqvist
+ *
+ * Authors:
+ * Mika T. Lindqvist <postmaster@raasu.org>
+ * Jun He <jun.he@arm.com>
+ *
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
```

Comment:
Bad copyright... For all code made by zlib-ng main contributors, the copyright should be assigned to zlib-ng project. This allows the file to be licensed under compatible license.

Reply:
I had just copied the copyright from fill_window_arm.c since that is where the code originally came from. What is the copyright I should use?

Reply:
For starters, any edit to the file should update the years in the last copyright. Any substantial change by new person should be added to authors list.

Reply:
I'm okay with leaving the copyright as it is, because I have not changed the code significantly.